tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -36,3 +39,97 @@ def test_fields(self):
         self.assertEqual(str(uut), "qwertzuiop")
         self.assertEqual(uut.marker, ("##", "#", "#"))
         self.assertEqual(uut.range, None)
+
+
+class PythonDocumentationCommentTest(unittest.TestCase):
+
+    Description = DocumentationComment.Description
+    Parameter = DocumentationComment.Parameter
+    ReturnValue = DocumentationComment.ReturnValue
+
+    def load_testdata(self, filename):
+        filename = (os.path.dirname(os.path.realpath(__file__)) +
+                    "/documentation_extraction_testdata/" + filename)
+        with open(filename, "r") as fl:
+            data = fl.read()
+
+        return data.splitlines(keepends=True)
+
+    def test_empty_docstring(self):
+        self.check_docstring("", [])
+
+    def test_description(self):
+        doc = " description only "
+        self.check_docstring(doc, [self.Description(desc=' description only ')])
+
+    def test_params_default(self):
+        self.maxDiff = None
+        doc = (" :param test:  test description1 \n"
+               " :param test:  test description2 \n")
+        expected = [self.Parameter(name='test', desc=' test description1 \n'),
+                    self.Parameter(name='test', desc=' test description2 \n')]
+        self.check_docstring(doc, expected)
+
+    def test_return_values_default(self):
+        doc = (" :return: something1 \n"
+               " :return: something2 ")
+        expected = [self.ReturnValue(desc='something1 \n'),
+                    self.ReturnValue(desc='something2 ')]
+        self.check_docstring(doc, expected)
+
+    def test_python_default(self):
+        data = self.load_testdata("default.py")
+
+        language = "python"
+        docstyle = "default"
+
+        docs = tuple(extract_documentation(data, language, docstyle))
+
+        parsed_docs = []
+
+        for doc in docs:
+            parsed_docs.append(doc.parse())
```

Comment:
oh you use `docs` also only once, let's do it like this:
```
parsed_docs = [doc.parse() for doc in extract_documentation(data, "python", "default")]
```
then you also don't need to collect the stuff from `extract_documentation` with a tuple beforehand :D